### ISSUE TITLE
Module x has no exported member

### ISSUE THREAD

**Issue by theunreal** [angular/material2#7625]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Don't fail build

#### What is the current behavior?
Can't build the app - material beta.10

using` ng build:`

Error:

ERROR in project
```
-dashboard/src/main/frontend/src/app/admin/applications/applications.module.ts (
12,9): Module '"C:/Users/eelnasi/Workspaces/asr1k/emb/integration/sd_avc_dca/con
troller-dashboard/src/main/frontend/node_modules/@angular/material/material"' ha
s no exported member 'MdTableModule'.
ERROR in project
-dashboard/src/main/frontend/src/app/core/toast.service.ts (2,9): Module '"C:/Us
ers/eelnasi/Workspaces/asr1k/emb/integration/sd_avc_dca/controller-dashboard/src
/main/frontend/node_modules/@angular/material/material"' has no exported member
'MdSnackBar'.
ERROR in project
-dashboard/src/main/frontend/src/app/shared/shared.module.ts (4,27): Module '"C:
project/
src/main/frontend/node_modules/@angular/material/material"' has no exported memb
er 'MdButtonModule'.
ERROR in project
-dashboard/src/main/frontend/src/app/shared/shared.module.ts (4,43): Module '"C:
project/
src/main/frontend/node_modules/@angular/material/material"' has no exported memb
er 'MdInputModule'.
ERROR in project
-dashboard/src/main/frontend/src/app/shared/shared.module.ts (5,3): Module '"C:/
Users/eelnasi/Workspaces/asr1k/emb/integration/sd_avc_dca/controller-dashboard/s
rc/main/frontend/node_modules/@angular/material/material"' has no exported membe
r 'MdCardModule'.
ERROR in project
-dashboard/src/main/frontend/src/app/shared/shared.module.ts (5,17): Module '"C:
project/
src/main/frontend/node_modules/@angular/material/material"' has no exported memb
er 'MdSnackBarModule'.
ERROR in project
-dashboard/src/main/frontend/src/app/shared/shared.module.ts (6,4): Module '"C:/
Users/eelnasi/Workspaces/asr1k/emb/integration/sd_avc_dca/controller-dashboard/s
rc/main/frontend/node_modules/@angular/material/material"' has no exported membe
r 'MdMenuModule'.
ERROR in project
-dashboard/src/main/frontend/src/app/shared/shared.module.ts (6,18): Module '"C:
project/
src/main/frontend/node_modules/@angular/material/material"' has no exported memb
er 'MdIconModule'.
```
.. etc for the rest of the material modules i'm using

#### What are the steps to reproduce?
run `npm install`

#### What is the use-case or motivation for changing an existing behavior?
Build fix

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 5 beta 7
Material beta.10
Typescript 2.4.2

#### Is there anything else we should know?
Tried to remove @angular/material and npm install

**Comment by crisbeto** [comment]
The `Md*` prefixed classes have been renamed to `Mat*` ([see the beta.11 docs](https://github.com/angular/material2/releases/tag/2.0.0-beta.11)). There's a [tool to help you migrate to the new prefixes.](https://www.npmjs.com/package/angular-material-prefix-updater)

**Comment by theunreal** [comment]
Hi @crisbeto, In my issues I wrote that I'm using beta.10.

Anyway, this problem occured because for some reasons material upgraded itself to version 12, seems like I messed up with the new npm version. the package.json declared material-10 while the package.lock declared material-12

Thanks

**Comment by crisbeto** [comment]
If you had something along the lines of `~2.0.0-beta.10` it might've updated it for you.

**Comment by sofiag** [comment]
Hello,
I'm having the same problem. Also using material-10. I tried to migrate as described in https://www.npmjs.com/package/angular-material-prefix-updater but I get the error: "Could not find updated prefix for identifier ..."  i.e. (MdDialog). Also checked the package-lock and indeed the version there is declared material-12. I tried changing it, but this still doesn't solve the problem. Any ideas of what I am missing?
Thanks
